jsapp/js/mixins.es6

```diff
@@ -213,6 +213,19 @@ mixins.dmix = {
       return this.props.uid;
     }
   },
+  // TODO
+  // Fix `componentWillUpdate` and `componentDidMount` asset loading flow.
+  // Ideally we should build a single overaching component that would
+  // handle loading of the asset in all necessary cases in a way that all
+  // interested parties could use without duplication or confusion and with
+  // indication when the loading starts and when ends.
+  componentWillUpdate(newProps) {
+    if (this.props.params.assetid !== newProps.params.assetid) {
```

Comment:
`newProps` will contain just `newForm` as its context when clicking "NEW" (adding a new form), making `newProps.params` return `undefined` and crashing here

Reply:
The visual bug you've discovered was actually caused by this crash ^ :)